test/browser/run-test.sh

```diff
@@ -34,9 +36,13 @@ echo "TEST_ALLOW_JOURNAL_MESSAGES: ${TEST_ALLOW_JOURNAL_MESSAGES:-}"
 echo "TEST_AUDIT_NO_SELINUX: ${TEST_AUDIT_NO_SELINUX:-}"
 
 RC=0
-test/common/run-tests --nondestructive --machine 127.0.0.1:22 --browser 127.0.0.1:9090 $EXCLUDES || RC=$?
+./test/common/run-tests \
+    --nondestructive \
+    --machine localhost:22 \
+    --browser localhost:9090 \
```

Comment:
Was the `--network=host` deliberate, i.e. does it simplify things significantly? Shouldn't this work equally well with `--machine _gateway:22`? Then it would isolate the test better.

Reply:
Great idea.  This is the kind of thing that obviously exists, if you think about it, but I had never thought about it.  I'll look into it.